tests/subprocesstest.py

```diff
@@ -0,0 +1,125 @@
+"""
+subprocesstest enables unittest test cases and suites to be run in separate
+python interpreter instances, in parallel.
+
+A base class SubprocessTestCase is provided that, when extended, will run test
+cases marked with @run_in_subprocess in a separate python interpreter.
+"""
+import unittest
+import subprocess
+import sys
+
+
+SUBPROC_TEST_ATTR = '_subproc_test'
+
+
+def run_in_subprocess(obj):
+    """
+    Marks a test case that is to be run in its own 'clean' interpreter instance.
+
+    When applied to a TestCase class, each method will be run in a separate
+    interpreter instance, in parallel.
```

Comment:
`s/, in parallel//`

From looking at this, that doesn't appear to be true anymore?

Reply:
Good catch